cobigen-maven/cobigen-maven-systemtest/src/test/resources/testdata/systemtest/Devon4JTemplateTest/TestEntityInputDataaccessGeneration/pom.xml

```diff
@@ -37,57 +37,57 @@
         <dependency>
             <groupId>com.devonfw.cobigen</groupId>
             <artifactId>templates-devon4j</artifactId>
-            <version>3.1.8</version>
+            <version>20.08.002</version>
         </dependency>
         <dependency>
           <groupId>com.devonfw.cobigen</groupId>
           <artifactId>javaplugin</artifactId>
-          <version>7.0.0-SNAPSHOT</version>
+          <version>7.1.0-SNAPSHOT</version>
```

Comment:
haven't we released the 7.1.0 now?

Reply:
please check all dependencies. snapshots are always not stable and tend to fail after the next release when the snapshots are cleaned up from the nexus. If possible make all dependency to stable versions